pkg/controller/render/render_controller.go

```diff
@@ -84,8 +85,10 @@ func New(
 	ctrl := &Controller{
 		client:        mcfgClient,
 		eventRecorder: eventBroadcaster.NewRecorder(scheme.Scheme, corev1.EventSource{Component: "machineconfigcontroller-rendercontroller"}),
-		queue:         workqueue.NewNamedRateLimitingQueue(workqueue.DefaultControllerRateLimiter(), "machineconfigcontroller-rendercontroller"),
 	}
+	ctrl.queue = workqueue.NewNamedRateLimitingQueue(workqueue.NewMaxOfRateLimiter(
+		&workqueue.BucketRateLimiter{Limiter: rate.NewLimiter(rate.Limit(5*time.Second), 1)},
```

Comment:
Can we make these rate limits a const with a descriptive comment about why e.g. 5 sec was chosen? like we do for `renderDelay` above